src/lib/nodes/doge/DogeNode.ts

```diff
@@ -12,12 +14,14 @@ import { BlockchainInfo } from './types/api/blockchain-info'
  * to the node and verify is status (online/offline, version, ping, etc.)
  */
 export class DogeNode extends Node<AxiosInstance> {
-  constructor(url: string) {
-    super(url, 'doge', 'node', NODE_LABELS.DogeNode)
+  constructor(endpoint: NodeInfo) {
+    super(endpoint, 'doge', 'node', NODE_LABELS.DogeNode)
   }
 
   protected buildClient(): AxiosInstance {
-    return createBtcLikeClient(this.url)
+    const baseURL = getBaseURL(this)
+
+    return createBtcLikeClient(baseURL)
   }
```

Comment:
We override the `baseURL` on each request anyway, so the instance can be create with a domain (this.url). Rollback the changes.

Reply:
The first request should always be made using the domain, so it's more correct to pass `this.url`.